woocommerce/woocommerce-gutenberg-products-block
# Product Gallery: fix CSS leak

## What

Fixes #11028. Currently, a style is leaking into the lightbox (dialog) Product Gallery from the regular gallery.

## Why



This PR fixes the style leaking. Also, it removes unnecessary style from the `editor.scss` of the Product Gallery Large Image.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. With a block theme, open `Appearance > Editor`.
2. Open `Templates > Single Product`.
3. Select `Convert to Blocks` if you are using the Classic Templates.
4. Add the `Product Gallery` block.
5. Ensure the `Next/Previous Buttons` are set to display **outside** of the image.
6. Open a product page on the frontend **with multiple product images** and click on the Large Image. Ensure that the zoom animation and arrow look okay.



## Screenshots or screencast



| Before | After |
| ------ | ----- |
|<video src=https://github.com/woocommerce/woocommerce-blocks/assets/4463174/bdfcd1f9-b496-4497-88e1-08bec7d01e82 />        |<video src=https://github.com/woocommerce/woocommerce-blocks/assets/4463174/1adaee2a-38ea-4e0a-bcc3-1cdab481f78c />|

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
